core/vm/jump_table.go

```diff
@@ -62,14 +62,6 @@ var (
 // JumpTable contains the EVM opcodes supported at a given fork.
 type JumpTable [256]operation
 
-func newYoloV1InstructionSet() JumpTable {
```

Comment:
This is because is not used and the lint failed?

Reply:
exactly, plus the whole Yolo thing is a test network for ethereum